RuntimeUnityEditor/RuntimeUnityEditorCore.cs

```diff
@@ -166,7 +166,7 @@ internal bool RemoveFeature(IFeature feature)
         internal static MonoBehaviour PluginObject => _initSettings.PluginMonoBehaviour;
         internal static ILoggerWrapper Logger => _initSettings.LoggerWrapper;
 
-        private static InitSettings _initSettings;
+        internal static InitSettings _initSettings;
```

Comment:
Instead of making this public it would be better to pass InitSettings as a parameter to CreateSkin()

Reply:
My Initial thought as well, but I couldn't do it as it's unity that is doing the call, so no way I can override the method signature to add the object settings, also I don't see why making it internal is a bad thing, since it's children are exposed internally anyway.